carbon-design-system/carbon
# chore(project): consolidate project deps

This PR consolidates several dependencies we have so that they use the same version. This reduces the total number of dependencies installed by the project since we are using only one version per dependency instead of several (ideally)

#### Changelog

**New**

**Changed**

- Consolidate the following deps:
  - `core-js`
  - `yargs`
  - `fast-glob`
  - `memfs`
  - `whatwg-fetc`
  - `sass`

**Removed**

#### Testing / Reviewing

- Verify build passes as expected
- Verify test passes as expected
- Verify storybook works as expected
